src/ensembl/src/shared/components/accordion/css/Accordion.scss

```diff
@@ -10,6 +10,9 @@
 }
 
 .accordionButtonDefault {
```

Comment:
I think there is a css issue here with the new Chevron button added to Accordion
![Screenshot 2021-07-01 at 11 54 37](https://user-images.githubusercontent.com/6347854/124113540-76855880-da63-11eb-8e63-1c96ffb7323e.png)